.github/workflows/deploy-autodokumentasjon-web.yaml

```diff
@@ -0,0 +1,40 @@
+name: "deploy: autodokumentasjon-web"
+'on':
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/deploy-autodokumentasjon-web.yaml
+      - autodokumentasjon-web/**
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy-to-cdn:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Install frontend
+        working-directory: autodokumentasjon-web
+        run: npm install
+
+      - name: Build frontend
+        working-directory: autodokumentasjon-web
+        run: npm run build
+
+      - uses: navikt/frontend/actions/spa-deploy/v1@main
+        with:
+          app-name: autodokumentasjon
+          team-name: pensjonsbrev
+          source: autodokumentasjon-web/dist
+          ingress: https://autodokumentasjon.intern.dev.nav.no
+          environment: dev
+          nais-deploy-apikey: ${{ secrets.NAIS_DEPLOY_APIKEY }}
```

Comment:
Vi har ikke deploy key lengre. vi bruker deploy v2 som bruker permissions fra nais-console for å deploye. https://console.nav.cloud.nais.io/team/pensjonsbrev/repositories